src/dashboard/Data/Browser/Browser.react.js

```diff
@@ -505,7 +505,7 @@ class Browser extends DashboardView {
           if (name === 'objectId' || this.state.isUnique && name !== this.state.uniqueField) {
             return;
           }
-          if (!!required) {
```

Comment:
I am always suspicious of that rule, because `!!` is not the same as removing it, if there are no type checks.

Reply:
Got you, but I believe this [rule](https://eslint.org/docs/rules/no-extra-boolean-cast) acts only in the cases in which `!!` makes no difference. I can't think of a scenario in which the rule would actually harm the logic.

It's not a problem to disable it, but you know, the less we customize the less we have to maintain.

Reply:
I think we can leave the rule, I checked and the double bang is only removed within `if` conditions where it's unnecessary. Let's hope the rule is intelligent enough to remove only the unnecessary ones, as you said.